src/library/process.js

```diff
@@ -110,18 +140,22 @@ export const processMessage = async (AWS, region, service, account,
       ...msgBody,
       payload: {
         ...msgBody.payload,
-        ...procRes.processResp,
+        ...procRes.processResp, // CR: Mickey: as noted else where, would love for this to be processResult.handlerResponse
       },
     },
     {
       ...msgAttribs,
-      status: procRes.status,
+      status: procRes.status, // CR: Mickey: [disregard: I see this is addressed in `neverThrowError`]
+      //    should there be a default value here or an error thrown? 
+      //    if procRes.status is undefined.  We probably don't want to publish SNSs without a status
       eventId: uuid(),
       emitter: service,
     },
   );
 
   // Delete the message from the queue using the rcptHandle, if available.
+  // CR: Mickey: What is the case where rcptHandle is not available?  When does that happen
+  //    and what happens if the message is not deleted from the queue?
```

Comment:
This is handled by the framework. rcptHandle is not pulled from the message in SQS for single requests as the response (200/ 201, 500) from the Lambda takes care of deletion or retention of the message in the SQS queue. However, for bulk requests, it is definitely pulled in the bulkHandler and delivered till here. For bulk requests Lambda is not triggered by SQS, but, Lambda wakes up on schedule and pulls messages from SQS queue, in which case, Lambda response will not have any effect on the deletion or retention of the messages in the queue.